request/index.d.ts

```diff
@@ -56,7 +56,7 @@ declare namespace request {
         delete(options: TUriUrlOptions & TOptions, callback?: RequestCallback): TRequest;
 
         forever(agentOptions: any, optionsArg: any): TRequest;
-        jar(): CookieJar;
+        jar(store?: any): CookieJar;
```

Comment:
It expects https://github.com/types/npm-tough-cookie/blob/master/lib/cookie.d.ts#L110
I'm not sure how to reference that one, so I've made that similar to https://github.com/alexey-pelykh/DefinitelyTyped/blob/master/request/index.d.ts#L83